prisma/schema.prisma

```diff
@@ -20,6 +20,14 @@ model User {
   personalAccessToken PersonalAccessToken[]
 }
 
+model PendingUser {
+  id                     Int      @id @default(autoincrement())
+  email                  String   @unique
+  password               String
+  accountActivationToken String?
```

Comment:
hmm, if we have decided to store the activation token, why is it optional?

I'm not sure if we need to store such token though in the database